src/unistd.rs

```diff
@@ -1644,6 +1644,15 @@ mod linux {
         Errno::result(res).map(drop)
     }
 
+    /// Replaces the current process image with a new one.
+    /// ([see exec(3)](http://man7.org/linux/man-pages/man3/exec.3.html))
```

Comment:
You need a blank line below this for the docs to look good. Please run `cargo doc --no-deps --open` and look at the `unistd` module to see the summary and click on `execvpe` to view the full description. This doesn't render well in the docs.